bpf/ep_config.h

```diff
@@ -54,6 +54,7 @@ DEFINE_U32(POLICY_VERDICT_LOG_FILTER, 0xffff);
 #define CUSTOM_CALLS_MAP test_cilium_calls_custom_65535
 #define LOCAL_DELIVERY_METRICS
 #define CONNTRACK_ACCOUNTING
+#define POLICYTRACING_ACCOUNTING
```

Comment:
Why do we define it by default? I think what we want is to not run __sync_fetch_and_add() by default.

Reply:
For what it's worth, this file is only used for compile testing and not used in production. However I think there is a solid debugability reason to keep this tracking stats on policymap by default and only optimize for performance for users who find such performance details to be critical to their application.